src/ddmd/dmodule.d

```diff
@@ -324,7 +324,7 @@ extern (C++) final class Module : Package
     int isDocFile;              // if it is a documentation input file, not D source
     bool isPackageFile;         // if it is a package.d
     int needmoduleinfo;
-
+    uint unitTestCounter;       // how many unittests habe been seen so far
```

Comment:
Haben -> Have.  And this field should be added to module.h too.

Reply:
Oops, fixing it.